core/contracts/financial-templates/implementation/FeePayer.sol

```diff
@@ -117,8 +117,9 @@ abstract contract FeePayer is Testable {
         emit RegularFeesPaid(regularFee.rawValue, latePenalty.rawValue);
 
         totalPaid = regularFee.add(latePenalty);
-        FixedPoint.Unsigned memory effectiveFee = totalPaid.divCeil(_pfc);
-        cumulativeFeeMultiplier = cumulativeFeeMultiplier.mul(FixedPoint.fromUnscaledUint(1).sub(effectiveFee));
+
+        // Add the adjustment to the cumulative fee multiplier by the fee paid and the current PFC.
```

Comment:
nit: I would not use "add" when describing how the CFM is adjusted, since its actually getting reduced by some amount proportional to the fee paid and the current PfC. Perhaps replace "Add the adjustment to...by" with "Reduce...proportionately to"